src/main/java/com/tg5/service/reports/ExcelAttendanceExportService.java

```diff
@@ -0,0 +1,66 @@
+package com.tg5.service.reports;
+
+import jakarta.servlet.ServletOutputStream;
+import jakarta.servlet.http.HttpServletResponse;
+import org.dhatim.fastexcel.Workbook;
+import org.dhatim.fastexcel.Worksheet;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.stereotype.Service;
+
+import java.io.IOException;
+import java.text.DateFormat;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.Map;
+
+@Service
+@Qualifier("excel")
+public class ExcelAttendanceExportService implements AttendanceExportService {
+    private Worksheet worksheet;
```

Comment:
I'm still not exactly sure about Java's threading stuff.. But since the class is a singleton, this can be written by multiple threads at once? So this should go as alocal variable.